src/Explorer/Controls/NotebookGallery/InfoComponent/InfoComponent.tsx

```diff
@@ -3,10 +3,14 @@ import { Icon, Label, Stack, HoverCard, HoverCardType, Link } from "office-ui-fa
 import { CodeOfConductEndpoints } from "../../../../Common/Constants";
 import "./InfoComponent.less";
 
-export class InfoComponent extends React.Component {
-  private getInfoPanel = (iconName: string, labelText: string, url: string): JSX.Element => {
+export interface InfoComponentProps {
+  onReportAbuseClick?: () => void;
+}
+
+export class InfoComponent extends React.Component<InfoComponentProps> {
+  private getInfoPanel = (iconName: string, labelText: string, url?: string, onClick?: () => void): JSX.Element => {
     return (
-      <Link href={url} target="_blank">
+      <Link href={url} target={url && "_blank"} onClick={onClick}>
```

Comment:
Does this force the browser to report errors if target is undefined? I'd check and confirm before this adds more noise in appinsights.

Reply:
This is just to support both opening a url in a new tab and have a custom click handler. In case where url is defined we want to set href and target, otherwise we just se onClick. Verified that it doesn't report any errors for undefined target (it is only expected when href is set).